src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -1202,10 +1266,12 @@
         sas=`az storage share generate-sas -n MyShare --account-name MyStorageAccount --https-only --permissions dlrw --expiry $end -o tsv`
         az storage file upload -s MyShare --account-name MyStorageAccount --source file.txt  --sas-token $sas
   - name: Generates a shared access signature for the share. (autogenerated)
-    text: az storage share generate-sas --account-key 00000000 --account-name MyStorageAccount --expiry 2037-12-31T23:59:00Z --name MyShare --permissions dlrw
+    text: |
```

Comment:
duplicate example names. it will be better to have different descriptions here.

Reply:
What do you suggest they do here?